content/documentation/pages/1-installation/3-kubernetes/1-helm.md

```diff
@@ -40,23 +46,9 @@ You should see the `Tiller` pod running.
 
 <!--TEMPLATE:https://raw.githubusercontent.com/helm/charts/master/stable/spring-cloud-data-flow/README.md-->
 
-#### Version Compatibility
-
-The following listing shows Spring Cloud Data Flow’s Kubernetes version compatibility with the respective Helm Chart releases:
-
-|                       | Chart 0.1.x | Chart 0.2.x | Chart 1.0.x | Chart 2.0.x |
-| --------------------- | :---------: | :---------: | :---------: | :---------: |
-| SCDF-K8S-Server 1.2.x |     ✅      |     ❌      |     ❌      |     ❌      |
-| SCDF-K8S-Server 1.3.x |     ❌      |     ✅      |     ❌      |     ❌      |
-| SCDF-K8S-Server 1.4.x |     ❌      |     ✅      |     ❌      |     ❌      |
-| SCDF-K8S-Server 1.5.x |     ❌      |     ✅      |     ❌      |     ❌      |
-| SCDF-K8S-Server 1.6.x |     ❌      |     ✅      |     ❌      |     ❌      |
-| SCDF-K8S-Server 1.7.x |     ❌      |     ❌      |     ✅      |     ❌      |
-| SCDF-K8S-Server 2.0.x |     ❌      |     ❌      |     ❌      |     ✅      |
-
 #### Expected output
 
-You should see the following output:
+After issuing the `helm install` command, you should see output similar to the following:
```

Comment:
Let's update the NOTE from "Get the application URL by running these commands:" --> "Get the Spring Cloud Data FLow's application URL by running these commands:"